german/es-wird-kalt/Kälte

```diff
@@ -0,0 +1,3 @@
+Die Beine werden langsam kalt. Sie wissen, dass Sie sich bewegen müssen.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.